core/tools/nginx/nginx.conf

```diff
@@ -195,9 +195,24 @@ http {
             try_files $uri $uri/ /index.html;
         }
 
+        location ~ ^/upload(/.*)$ {
+            client_max_body_size 100M;
+            alias /usr/blueos$1;
```

Comment:
maybe.. /usr/blueos/frontend ?

Reply:
the current way allows us to use /userdata as a path, which was my goal